x-pack/filebeat/processors/decode_cef/decode_cef_test.go

```diff
@@ -315,14 +315,8 @@ func assertEqual(t testing.TB, expected, actual interface{}) bool {
 	expJSON, _ := json.MarshalIndent(expected, "", "  ")
 	actJSON, _ := json.MarshalIndent(actual, "", "  ")
 
-	diff, _ := difflib.GetUnifiedDiffString(difflib.UnifiedDiff{
-		A:        difflib.SplitLines(string(expJSON)),
-		B:        difflib.SplitLines(string(actJSON)),
-		FromFile: "Expected",
-		ToFile:   "Actual",
-		Context:  1,
-	})
-	t.Errorf("Expected and actual are different:\n%s", diff)
+	t.Errorf("Expected and actual are different:\n%s",
+		cmp.Diff(string(expJSON), string(actJSON)))
```

Comment:
In the logging I add some context to ensure that people know which is `got` and which is `want`. For example [here](https://github.com/elastic/beats/blob/bfc1b119c94fcdb9258594e32b9c3d71ac9f8664/libbeat/processors/cache/file_store_test.go#L441-L442).

Reply:
Why was this resolved without action?

Reply:
@AndersonQ could you have a look please?

Reply:
sorry, I replied but I forgot to post the comments. That's why I resolved, you did not marke the PR as needing changes nor said it was a blocker, therefore I understood it was just a suggestion, besides I ~thought I~ explained why I don't think the change is necessary.

Reply:
> I believe it's already clear:
> 
> ```
> Expected and actual are different:
>           string(
>         - 	"want",
>         + 	"got",
>           )
> ```

the comment I forgot to post

Reply:
In the context of a diff, `-` and `+` are intrinsically meaningful since a diff is a directed change. Diff output has been bent to render test mismatch, IMO there is not an unambiguous mapping between the diff syntax and the direction of the test mismatch (I have seen people use both, and go-cmp uses non-conventional direction for the args to `Diff` in the context of testing — conventional in the context of diff changes).